corehq/apps/integration/payments/const.py

```diff
@@ -21,3 +23,10 @@ class PaymentProperties(str, Enum):
 
 PAYMENT_SUCCESS_STATUS_CODE = 202
 PAYMENT_SUBMITTED_DEVICE_ID = 'momo_payment_service'
+
+
+class PaymentStatus(models.TextChoices):
+    NOT_REQUESTED = 'not_requested', _("Not requested")
+    PENDING = 'pending', _("Pending")
```

Comment:
I am confused by Not requested and pending, how are they different?
May be good to add a docstring to this for those not familiar with the payment workflow.

Reply:
Yeah, when I coded it it made sense but since we have the "verified" / "not verified" filters it's not relevant anymore.